src/web.js

```diff
@@ -67,8 +71,9 @@ const httpServer = function httpServer(options: OptionsType) {
   app.use(compression());
   app.set('view engine', options.viewEngine || 'pug');
   app.use(logger('dev'));
-  app.use(bodyParser.json());
-  app.use(bodyParser.urlencoded({extended: true}));
+
+  app.use(bodyParser.json(options.bodyParser && options.bodyParser.jsonOpts ? options.bodyParser.jsonOpts : {}));
+  app.use(bodyParser.urlencoded(options.bodyParser && options.bodyParser.urlencodedOpts ? options.bodyParser.urlencodedOpts : {extended: true}));
```

Comment:
`options.bodyParser.urlencodedOpts` should probably be `Object.assign({extended: true}, options.bodyParser.urlencodedOpts)`